ynr/apps/candidates/static/js/person_form.js

```diff
@@ -164,6 +165,12 @@ var populate_party_selects = function() {
   allPartySelects.attr("disabled", false);
   $(PARTY_WIDGET_INPUT_CLASS).hide();
   allPartySelects.each(setup_single_party_select);
+
```

Comment:
@michaeljcollinsuk Just to confirm, this should be a multiple or single select element?